### ISSUE TITLE
Anmeldung mit Studiengang Lehramt funktioniert nicht

### ISSUE THREAD

**Issue by larsgroeber** [PhysikOnline-FFM/po-fp-backend#49]
**Response**
```
{
  "institutes": [
    "Es wurden mehr als 2 oder keine Institute angegeben [OrderedDict([('id', 11), ('name', 'PI'), ('places', 2), ('graduation', 'L'), ('semesterhalf', 3), ('notes', None)])]."
  ]
}
```
**Request**
```
POST: https://po-fp-staging.physikelearning.de/api/user_registrant/
{
  "user_firstname": "Lars",
  "user_lastname": "Gr\u00f6ber",
  "user_login": "s0053274",
  "user_mail": "email@email.com",
  "user_matrikel": "doesNotExist",
  "graduation": "L",
  "notes": "",
  "institutes": [
    {
      "id": 11,
      "name": "PI",
      "graduation": "L",
      "places": 2,
      "semesterhalf": 3,
      "notes": null
    }
  ],
  "status": null,
  "partner": null
}
```

**Comment by chgad** [comment]
This issue is caused by the validation of the data provided. For now the serializer expects there are exactly 2 institutes. Of course we do need the option for students with "Lehramt" to provide only 1 institute.

**Comment by chgad** [comment]
Fixed via #51 .
